### PR TITLE
added server.js, set up heroku, fixed package.json

### PR DESCRIPTION
**Pull request by FileExplorer**
Checkout the [Engineer Workflow Doc](https://docs.google.com/presentation/d/1pdzuHG-yO7_qW6jiByaGTLztKZ6b72lBNqSdNT5WZQ8/edit) to make sure you're contributing correctly! :)

**[Tickets](https://trello.com/b/dYOQKJGv/reclass)** 
Trello Ticket: SETUP: Heroku hook to ToyNet master GItHub Branch
Additional Notes: server.js serves up the build folder. One of the issues with uploading the repo directly to heroku is that heroku uses a random port number every time it launches, however with react, react has a built-in server.js (you run it using npm start) and that forces port 3000. This will cause heroku to break and give a H10 error. All server.js does is either take port 8080 by default or if given a port (by heroku in this case) use that instead.

**[Dependencies](https://docs.google.com/document/d/1CM6OA2nlhI3DHnHdDEgySukx_tZ_wsG-C2YgraZRvJA/edit#heading=h.czpk4cbi7z4g)**

**[Explain Impact on DB](https://docs.google.com/document/d/1CM6OA2nlhI3DHnHdDEgySukx_tZ_wsG-C2YgraZRvJA/edit#heading=h.285lgcs4z84o)**

**[Testing](https://docs.google.com/document/d/1CM6OA2nlhI3DHnHdDEgySukx_tZ_wsG-C2YgraZRvJA/edit#heading=h.rs692kr58cf8)**

How did you test this? List of actions & expected behavior with helpful screenshots for reviewer:

If this change is performance related, please document what tests were / should be done to verify improvement:

### CODE REVIEW

**Comment by sammytran277** [comment]
I think I need a bit more context about what is going on here. It looks like you've added Express as a dependency and wrote some basic server-side code in the server.js file. Why do we need this in toynet-react?

**Comment by sammytran277** [comment]
Also, for future PRs, I think you should fill out Tay's PR template.

**Comment by FileExplorer** [comment]
@sammytran277  server.js serves up the build folder. One of the issues with uploading the repo directly to heroku is that heroku uses a random port number every time it launches, however with react, react has a built-in server.js (you run it using npm start) and that forces port 3000. This will cause heroku to break and give a H10 error. All server.js does is either take port 8080 by default or if given a port (by heroku in this case) use that instead.

Also, where can I find Tay's PR template?

**Comment by takakonishimura** [comment]
@FileExplorer Can you add a comment to server.js file itself explaining that it can be deleted when we move off heroku?

File: server.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const express = require('express');
```
Comment (takakonishimura) [inline server.js:1]: Comment here

**Pull request merged**